reduction/imaging_parameters.py

```diff
@@ -2798,15 +2798,10 @@
     for field in allfields
     for band in ("B3", "B6")
     for array in ("12M", "7M12M", "7M")
-    # for robust in (0,)
     for robust in (-2, 0, 2)
     for contsub in ("", "_contsub")
 }
 
-for key in line_imaging_parameters_default:
-    line_imaging_parameters_default[key]['scales'] = [0,5,15,45]
-    line_imaging_parameters_default[key]['threshold'] = "10sigma"
-    line_imaging_parameters_default[key]['niter'] = 5000 # try to avoid divergence
```

Comment:
This was definitely a mistake, and should be:
```python
for key in line_imaging_parameters_default:
    if '7M12M' in key:
        line_imaging_parameters_default[key]['scales'] = [0,5,15,45]
        line_imaging_parameters_default[key]['threshold'] = "10sigma"
        line_imaging_parameters_default[key]['niter'] = 5000 # try to avoid divergence
```

Are you good with adding that back?  It was definitely an error as written.

Reply:
I see. Will change today later.